src/AzureIoTHubClient.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Arduino. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <time.h>
+#include <sys/time.h>
+
+#include <AzureIoTHubClient.h>
+
+Client* AzureIoTHubClient::sslClient = NULL;
+
+AzureIoTHubClient::AzureIoTHubClient(Client& sslClient)
+{
+    AzureIoTHubClient::sslClient = &sslClient;
+}
+
+void AzureIoTHubClient::begin()
+{
+#if defined(ARDUINO_ARCH_ESP8266)
+    Serial.setDebugOutput(true);
```

Comment:
I think the api only exists in esp8266 board so we would need to have it commented out like the other esp8266 specific lines. And yes, if set to true on esp there should be wifi diagnostic stuff. 

Some info on it here in the Serial section: https://github.com/esp8266/Arduino/blob/master/doc/reference.md